http-server/src/main/java/org/triplea/server/lobby/game/listing/GameListingEventQueue.java

```diff
@@ -25,36 +21,19 @@
   private final BiConsumer<Collection<Session>, ServerMessageEnvelope> broadcaster;
 
   @Getter(value = AccessLevel.PACKAGE, onMethod_ = @VisibleForTesting)
-  private final Map<String, Session> sessions = new HashMap<>();
+  private final SessionSet sessionSet;
```

Comment:
As mentioned in #5802 we need a way to have a concurrent SessionSet

Reply:
Use concurrent set: c664a7f5e

Additional thread safety updates: aa8f957bc (message broadcaster receives other collections from more locations than here)